pickee_api/gateway.py

```diff
@@ -15,24 +16,6 @@
 TMDB_ITEMS_PER_PAGE = 20
 
 
-def tmdb_example_endpoint(request):
```

Comment:
These endpoints should be removed from the urls.py too. I'm having an error trying to run the code

Reply:
thanks